homeassistant/components/zha/__init__.py

```diff
@@ -193,7 +193,7 @@ async def async_migrate_entry(
             data[CONF_RADIO_TYPE] = "znp"
 
         config_entry.version = 3
```

Comment:
Let's give this one more try 😆. Is *this* the correct way to modify `config_entry` during a migration (including changing the `version`?)